clang/lib/Sema/CheckExprLifetime.cpp

```diff
@@ -535,6 +535,9 @@ static void visitFunctionCallArguments(IndirectLocalPath &Path, Expr *Call,
 
   bool EnableGSLAnalysis = !Callee->getASTContext().getDiagnostics().isIgnored(
       diag::warn_dangling_lifetime_pointer, SourceLocation());
+  bool EnableDanglingCapture =
+      !Callee->getASTContext().getDiagnostics().isIgnored(
+          diag::warn_dangling_reference_captured, SourceLocation());
```

Comment:
I would not disable this based on new warning because:
1. This is not the warning that is triggered in this path.
2. We are considering both annotations as aliases in case of constructors. It would be better to do this unconditionally.

Reply:
Done. I added some comments to clarify the behavior.

Reply:
Thanks.